modules/mesaides/files/ma-web.conf

```diff
@@ -11,4 +11,11 @@ setuid main
 
 chdir /home/main/mes-aides-ui
 
-exec node server
+script
```

Comment:
A single `exec` command can be specified directly but multiple commands have to be wrapped in a `script` section, cf. http://upstart.ubuntu.com/wiki/Stanzas#exec.